### PR TITLE
[WIP]Fixes golint issues in maya/types/profiles/v1/profiles.go

### PR DESCRIPTION
**Pull request by pandyamarut**
This PR Adds the missing comment from the file. (File path given)

: 
fixes #2133
Signed-off -by: Marut pandya <pandyamarut@gmail.com>

### CODE REVIEW

File: types/profiles/v1/profiles.go [hunk 1]
```diff
@@ -87,7 +89,7 @@ const (
 // USAGE:
 //  Already mentioned in PlacementPolicyKey
 type PlacementPolicyValue string
-
+//SpreadReplicaPPV holds placement policy value for spread-replica.
 const (
 	SpreadReplicaPPV PlacementPolicyValue = "spread-replica"
 	StickyReplicaPPV PlacementPolicyValue = "sticky-replica"
```
Comment (princerachit) [inline types/profiles/v1/profiles.go]: Possible linting issue with this constant. Please write comments for each constant

File: types/v1/quantity.go [hunk 2]
```diff
@@ -13,7 +13,8 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-// This is extracted from k8s.io/apimachinery/pkg/api/resource/quantity.go
+// Package v1 is extracted from k8s.io/apimachinery/pkg/api/resource/quantity.go
+
```
Comment (princerachit) [inline types/v1/quantity.go]: new line not required

**Comment by pandyamarut** [comment]
@princerachit please review.

File: types/profiles/v1/profiles.go [hunk 3]
```diff
@@ -13,11 +13,9 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 */
-
-// This file deals with the labels, constants, defaults & utility
+// Package v1 deals with the labels, constants, defaults & utility
```
Comment (Akash4927) [inline types/profiles/v1/profiles.go]: Can we provide a new line instead of providing a comment for package?

**Comment by sonasingh46** [comment]
@pandyamarut 
Closing due to inactivity. Please let us know if we need to work on the same.

**Pull request closed without merging**